content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -23,6 +23,9 @@ deploy Kubeflow on Amazon Web Services (AWS).
 
 You do not need to have an existing Amazon Elastic Container Service for Kubernetes (Amazon EKS) cluster. The deployment process will create a cluster for you.
 
+The install tools uses `eksctl` command, and doesn't support `--profile` option, now.
```

Comment:
Please would you add a link to "`--profile`" so readers know which command this parameter belongs to.

Reply:
I couldn't find the description about '--profile' in the official document(https://eksctl.io/) and AWS websites.
But there is the option at current `eksctl` command.
I could find several documents or issues about it, but I think these pages aren't clear enough to link.

https://www.weave.works/blog/getting-started-with-openfaas-kubernetes-operator-on-eks
https://github.com/weaveworks/eksctl/issues/57

Reply:
Makes sense, thanks!